pkg/agent/qrm-plugins/network/staticpolicy/util.go

```diff
@@ -25,18 +25,30 @@ import (
 
 	"github.com/kubewharf/katalyst-api/pkg/consts"
 	"github.com/kubewharf/katalyst-core/cmd/katalyst-agent/app/agent"
+	"github.com/kubewharf/katalyst-core/pkg/agent/qrm-plugins/network/state"
 	"github.com/kubewharf/katalyst-core/pkg/util/general"
 	"github.com/kubewharf/katalyst-core/pkg/util/machine"
 )
 
 type NICFilter func(nics []machine.InterfaceInfo, req *pluginapi.ResourceRequest, agentCtx *agent.GenericContext) []machine.InterfaceInfo
 
 var nicFilters = []NICFilter{
-	filterNICsByAvailability,
 	filterNICsByNamespaceType,
 	filterNICsByHint,
 }
```

Comment:
We may not need them any more. I will remove them later if the existing implementation is correct.